modules/ROOT/pages/prerequisites/index.adoc

```diff
@@ -56,61 +60,117 @@ Note when referencing software builds for a particular architecture, *ARM* stand
 === RAM Considerations
 
 // harvested from https://owncloud.dev/architecture/efficient-stat-polling/
+// text adopted based on an intense discussion with jfd/willy on 11.3.
+
+// fixme: how to read stat info (local and eg nfs)
+
+RAM requirements are mainly based on caching the mounted filesystems meta-data like inodes (even extended attributes are supported) and include calculated ETags (entity tag). Meta-data caching is mostly based on the kernal caching capability of the OS for the mounted filesystems. Dependent on the filesystem(s) mounted, a different block size for the meta-data may be used. Avoid swap space usage because of meta-data caching for performance reasons.
```

Comment:
```suggestion
RAM requirements are mainly based on caching the mounted filesystems meta-data like inodes (even extended attributes are supported) and include calculated ETags (entity tag). Meta-data caching is mostly based on the kernel caching capability of the OS for the mounted filesystems. Dependent on the filesystem(s) mounted, a different block size for the meta-data may be used. Avoid swap space usage because of meta-data caching for performance reasons.
```